pkgs/by-name/jn/jnv/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, autoconf
+, automake
+, clang
+, libclang
+, libtool
+}:
+rustPlatform.buildRustPackage rec
+{
```

Comment:
Please read CONTRIBUTING.md before marking *Fits CONTRIBUTING.md*

https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax

Reply:
Apologies it was my nix formatter